plugin/src/test/java/io/jenkins/plugins/casc/SchemaGenerationTest.java

```diff
@@ -54,9 +63,32 @@ public void schemaShouldSucceed() {
                         "      \"$ref\": \"#/definitions/" + a.type.getName() + "\"\n" +
                         "    },");
 
+
             }
         }
 
         System.out.println(schemaString);
+
+        /**
+         * Used to generate the schema for the descriptors
+         */
+        ConfigurationAsCode configurationAsCode = ConfigurationAsCode.get();
+
+        configurationAsCode.getConfigurators()
+                .stream()
+                .forEach(configurator -> {
+                    if(configurator.getClass().getSimpleName().equals("HeteroDescribableConfigurator")) {
```

Comment:
@timja Probably we could do this for every object :) If there is a better way do suggest it

Reply:
something like:
```suggestion
                    if (configurator instanceof HeteroDescribableConfigurator.class)) {
```

eventually we would like to lookup the schema generation by the configurator type, but not needed yet, that would allow other people to contribute schema generators from a custom configurator